example/examples/Rebind.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { View, Button } from 'react-native';
+
+import WebView from 'react-native-webview';
+
+export default class Rebind extends Component {
+  constructor(props) {
+    super(props);
+
+    this.googleRef = React.createRef(null);
+    this.yelpRef = React.createRef(null);
+  }
+
+  render() {
+    return (
+      <View style={{ height: 400 }}>
+        <View style={{ width: 300, height: 200, borderWidth: 2, borderColor: 'red' }}>
+        <WebView
+          ref={this.googleRef}
+          webViewKey="TEST"
```

Comment:
nit: Should we put "TEST" in a const, since it's used in four places in this file?